test/Writer/EntryTest.php

```diff
@@ -767,10 +767,13 @@ public function testEntryWriterEmitsNoticeDuringFeedImportWhenGooglePlayPodcastE
             'messages' => [],
         ];
 
-        /** @psalm-suppress UnusedClosureParam */
-        set_error_handler(static function (int $errno, string $errstr) use ($notices): void {
+        // phpcs:disable WebimpressCodingStandard.NamingConventions.ValidVariableName.NotCamelCaps
+        set_error_handler(static function (int $_errno, string $errstr) use ($notices): ?bool {
```

Comment:
Is the underscore to mark is as unused?

Reply:
Yup, that's the convention in psalm and PHPStan 👍